lib/dealership.rb

```diff
@@ -0,0 +1,68 @@
+# require './lib/car'
+#
+#
+#
+class Dealership
+
+  attr_reader :name, :address, :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @total_val = 0
+    @inv_by_cost = []
+  end
+
+  def inventory_count
+    @inventory.count
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    @inventory.count > 1
```

Comment:
I believe this should be `@inventory.count > 0` meaning there is at least 1 car. I may also use `inventory_count` here.